x/oracle/types/events.go

```diff
@@ -0,0 +1,6 @@
+package types
+
+const (
+	EventTypeRegister         = "oracle_registered"
```

Comment:
I think it would be better to make this event type much clearer.
"oracle_registered" seems to be meant that "oracle is registered. so what do you do next?" to me.

This event is emitted when a request for oracle registration is submitted.
Thus, I suggest to rename it something like `EventTypeRegisterRequested` or `EventTypeRegisterSubmitted`. (Actually I don't like this name since it's too long.. but I have no better idea 😭)

Reply:
The event type was renamed as we discussed in #391.